### ISSUE TITLE
No shebang in conditional crashes Munki

### ISSUE THREAD

**Issue by waderobson** [munki/munki#598]
Hello. I ran into this yesterday when I sent out a half-baked conditional to some of the fleet. 
To recreate this remove the top line of a conditional script. IOW no `#!/bin/bash` or `#!/usr/bin/python`.

```
Starting...
    Performing preflight tasks...
ERROR: Unexpected error in updatecheck:
Traceback (most recent call last):
  File "/usr/local/munki/managedsoftwareupdate", line 1048, in <module>
    main()
  File "/usr/local/munki/managedsoftwareupdate", line 742, in main
    updatecheckresult = updatecheck.check(client_id=options.id)
  File "/usr/local/munki/munkilib/updatecheck.py", line 2973, in check
    munkicommon.getConditions()
  File "/usr/local/munki/munkilib/munkicommon.py", line 2454, in getConditions
    utils.runExternalScript(conditionalscriptpath))
  File "/usr/local/munki/munkilib/utils.py", line 131, in runExternalScript
    stderr=subprocess.PIPE)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 1335, in _execute_child
    raise child_exception
OSError: [Errno 8] Exec format error
```

I was able to catch it by doing this. 
https://gist.github.com/waderobson/36461224cd8ec8dad60e4315c7c4faf0#file-gistfile1-txt-L29-L40

Should I create a PR?

**Comment by gregneagle** [comment]
Yes.

**Comment by gregneagle** [comment]
Addressed in #599.
